js/validation.js

```diff
@@ -0,0 +1,36 @@
+// 이메일 유효성 검사
+function emailCheck(email_address){     
+  let email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+  if(!email_regex.test(email_address)){ 
+      return false;
+  } else{
+      return true;
+  }
```

Comment:
```suggestion
  return email_regex.test(email_address) 
```
이미 boolean 값이 리턴되고 있어서 요렇게만 써도 될것같아요